hicommonwealth/commonwealth
# [ERC20 Launchpad] - UI for Launch token community step

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8706

## Description of Changes
- Added UI for Launch token community step of tokenized community creation flow

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_TOKENIZED_COMMUNITY`
- Visit `/createTokenCommunity`
- Fill the form - verify the form validation works as expected
- Submit
- Verify you get redirected to step 3 "Sign and Launch" with a "Not Implemented" label
- Verify there are no regressions with the normal community creation flow

## Deployment Plan
N/A

## Other Considerations
N/A
